model/oracledb/registry.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - id: registry.oracledb
+    type: attribute_group
+    display_name: Oracle Database Attributes
+    brief: >
+      This section defines attributes for Oracle Database.
+    attributes:
+      - id: oracledb.implicit_release
```

Comment:
system-specific attribute namespace needs to match the `db.system.name` identifier (i.e. it has to be `oracle.db`

Reply:
Yes modified `oracledb.implicit_release `to `oracle.db.implicit_release`

similarly changed for other attributes too.